lib/takeaway.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'dish.rb'
+require_relative 'text.rb'
+require 'twilio-ruby'
+
+class Takeaway 
```

Comment:
May be an idea to split Takeaway class into separate classes to have single responsibility